tests/gnrc_dhcpv6_client/tests-with-config/check-config.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/sh
+#
+# check_config.sh
+# Copyright (C) 2021 Martine Lenders <mail@martine-lenders.eu>
+#
+# Distributed under terms of the MIT license.
+#
+
+ip link show dev "${IFACE}" > /dev/null
```

Comment:
i think this is the first of its kind i would like to suggest that in case of error something helpful is printed 

cloud not find IFACE: ${IFACE}, initialsation may be needed

Reply:
Done. The output now looks like this:

```
Device "tapbr0" does not exist.
You can create "tapbr0" by using e.g. `dist/tools/tapsetup/tapsetup`
System configuration for tests is not available.
```

Reply:
I sometimes use eth0 (or the current fancy naming sceme for this ) -> "You can create" -> "You may be able to create"